elado/malom/index.md

```diff
@@ -3,4 +3,56 @@ layout: elado
 title: Eladó malom
 ---
 
-#![Malom](http://upload.wikimedia.org/wikipedia/commons/0/02/Campo_de_Criptana_Spain_Windmill.jpg) Eladó malom
+#![Kívülről](http://i.imgur.com/cHwgdIm.jpg) Eladó önfenntartó tanya a Bakonyban!
+
+## Irányára: 135000000 HUF
+
+Izgalmas, különleges, összesen 5,8 ha-os ingatlant ölel a Bakonyban eredő bő vizű Séd patak két ága, Balatonfüredtől mindössze 30 km-re!
+
+A Szentgáli Református Egyház megbízásából 1854-ben épült vízimalmot új tulajdonosai 2009-ben kezdték felújítani Herend külterületén. Hagyománytisztelő ízléssel a mai kor igényei szerint is kényelmes, igényes otthont teremtettek a történelmünkre emlékező falak között. 
+
+A 300 m2-es, hat szobás, kétszintes épületben egy nagy lakást és vendégek számára egy kényelmes, kétszobás apartmant alakítottak ki. A tetőtér beépítése még nem fejeződött be.
+
+![Kívülről](http://i.imgur.com/JjwSV0l.jpg)
+
+A gyönyörű és egészséges környezetben álló otthon kiemelkedő előnye, hogy a fenntartása nem csak környezetbarát, minden szolgáltatótól független, de teljesen ingyenes is! A malom kerekét a 3 m-ről lezúduló patakvíz hajtja. Az elektromos energia egy része innen származik, a tetőn pedig napkollektorok és napelemek termelik az energiát. A bevizsgált ivóvizet a saját kút adja, a fűtéshez a saját 0,8 ha-os erdő biztosítja a tüzelőt. Az otthon melegét a központi fűtés mellett a cserépkályhában és kandallóban pattogó tűz hangulata is fokozza…
+
+![Kívülről](http://i.imgur.com/amgPlmg.jpg)
+
+A közel 4,5 ha-os jó minőségű szántóból most 1,2 ha-on a gazdáikat sajttal ajándékozó kecskék legelnek. 6100 m2 kivett területen áll a lakóház, udvar, gazdasági épület, mellettük a jól termő zöldséges kert és két kicsi, összesen 250 m2-nyi tavacska, amelyek tiszta vizében pisztrángok élnek. A kerék hajtására és a tavak létesítésére jelenleg érvényes vízjogi engedélyt csak 2028-ban kell hosszabbítani. 
+A terület két oldalán a természetes határ a Séd két ága, így engedélyeztetés után könnyedén kialakítható a mostani legelő területén egy kb. 1 ha-nyi gravitációsan tölthető halastó, sőt akár több tó is.
+Az eladó terület közvetlen szomszédságában további termőföld területek is vásárolhatók.
+
+![Kívülről](http://i.imgur.com/Nae804W.jpg)
+
+![Kívülről](http://i.imgur.com/96RdIM1.jpg)
+
+![Kívülről](http://i.imgur.com/iVekA8C.jpg)
+
+Herend (http://hu.wikipedia.org/wiki/Herend) Veszprémtől 15 km, Balatonfüredtől 30 km, Budapesttől 120 km távolságra van.
```

Comment:
Most vettem csak észre, hogy ez a rendszer nem csinál automatikusan linket a weblap címekből. Két lehetőség van:

 - Ha kacsacsőrök közé teszed a címet (`<http://.../Herend>`), akkor megjelenik a szövegben, de klikkelhető, mint egy link. https://github.com/darabos/megingat/commit/59cdd2643455ae69995bf496d8b1fe6f3587bc8a
 - Lehet olyat, hogy egy szövegrészt szeretnél linkelni egy weboldalra. Ehhez a szövegrészt szögletes zárójelbe kell tenned és rögtön utána a címet sima kerek zárójelbe: `[Herend](http://.../Herend)` https://github.com/darabos/megingat/commit/18aa16a22c9ea094ebae2869d37ac2d891655035

Most meg is csináltam ezt a második fajtát. De javítsd át nyugodtan, ha a másikat szeretnéd.